src/commands/dao_cmds/token.js

```diff
@@ -1,26 +1,9 @@
-const {
-  manifestMiddleware,
-  moduleMiddleware,
-  environmentMiddleware,
-} = require('../../middleware')
-
-const MIDDLEWARES = [
-  manifestMiddleware,
-  moduleMiddleware,
-  environmentMiddleware,
-]
-
 exports.command = 'token <command>'
 
 exports.describe = 'Create and interact with MiniMe tokens'
 
 exports.builder = function(yargs) {
-  return yargs
-    .commandDir('token_cmds', {
-      visit: cmd => {
-        cmd.middlewares = MIDDLEWARES
-        return cmd
-      },
-    })
-    .demandCommand(1, 'You need to specify a command')
+  const cmd = yargs.commandDir('token_cmds')
+  cmd.demandCommand(1, 'You need to specify a command')
+  return cmd
```

Comment:
To make this a bit cleaner, since `yargs` always uses the builder pattern, is:

```js
return yargs
  .commandDir('token_cmds')
  .demandCommand(1, 'You need to specify a command')
```